<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetBGColorDialog</class>
 <widget class="QDialog" name="SetBGColorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置背景颜色</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>31</y>
     <width>292</width>
     <height>118</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>R</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="5" rowspan="2">
     <widget class="QLCDNumber" name="RlcdNumber"/>
    </item>
    <item row="1" column="1" colspan="4">
     <widget class="QSlider" name="rSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="2" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>G</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="5" rowspan="2">
     <widget class="QLCDNumber" name="GlcdNumber"/>
    </item>
    <item row="3" column="1" colspan="4">
     <widget class="QSlider" name="gSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0" rowspan="2" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>B</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="5" rowspan="2">
     <widget class="QLCDNumber" name="BlcdNumber"/>
    </item>
    <item row="5" column="1" colspan="4">
     <widget class="QSlider" name="bSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SetBGColorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SetBGColorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RlcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>GlcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BlcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
